src/template.lib.php

```diff
@@ -81,6 +81,22 @@ private static function replacePseudoVars($content = '', $params = array())
             }
         }
 
+        global $locale;
+
+        if ( $locale !== 'en_US' ) {
+            require_once __DIR__ . '/strings.php';
+
+            preg_match_all('~{{(.+)}}~', $content, $matches);
+
+            if ( ! empty( $matches[1] ) ) {
+                foreach($matches[1] as $string) {
+                    $pattern = sprintf('~{{%s}}~', preg_quote($string));
```

Comment:
It should be `preg_quote($string, '~')`

Reply:
Fixed in https://github.com/Sucuri/sucuri-wordpress-plugin/pull/72/commits/ab7bec3b77251e800e2ea9f6cb9aceaa2aa1b50b